lib/utils/user.js

```diff
@@ -39,8 +39,7 @@ export const loggedInUserHash = _.once(async (): Promise<?string> => {
 });
 
 export const loggedInUserInfo = _.once((): Promise<CurrentRedditUser | void> =>
-	!isLoggedIn() ? Promise.resolve() : fetch('/api/me.json', { credentials: 'include' })
-		.then(r => r.json())
+	!isLoggedIn() ? Promise.resolve() : ajax({ url: '/api/me.json', type: 'json' })
```

Comment:
There appear to be no other instances of `fetch` in `lib/modules/**` and I'm sorry I didn't use `ajax` in the first place.